### ISSUE TITLE
android build crash

### ISSUE THREAD

**Issue by Nader2004** [LeGoffMael/video_editor#37]
Hello there. Whenever is add this package , the build always fails with this error : 

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugDuplicateClasses'.
> 1 exception was raised by workers:
  java.lang.RuntimeException: Duplicate class com.arthenica.mobileffmpeg.Abi found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.AbiDetect found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.AsyncFFmpegExecuteTask found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.AsyncFFprobeExecuteTask found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.AsyncGetMediaInformationTask found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.BuildConfig found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.CameraSupport found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
Duplicate class com.arthenica.mobileffmpeg.Config$1 found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.ExecuteCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.FFmpeg found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.FFmpegExecution found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.FFprobe found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.GetMediaInformationCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.Level found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.LogCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.LogMessage found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.MediaInformation found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
Duplicate class com.arthenica.mobileffmpeg.MediaInformationParser found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.Packages found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.Signal found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.Statistics found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.StatisticsCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.StreamInformation found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.util.Pair found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
  Duplicate class com.arthenica.mobileffmpeg.util.Trio found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 31m 56s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)

```
I have followed already the installation steps on Android

**Comment by LeGoffMael** [comment]
please try again with latest 1.2.3

**Comment by youlast** [comment]
Hey , did you find a solution?


> Hello there. Whenever is add this package , the build always fails with this error :
> 
> ```
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Execution failed for task ':app:checkDebugDuplicateClasses'.
> > 1 exception was raised by workers:
>   java.lang.RuntimeException: Duplicate class com.arthenica.mobileffmpeg.Abi found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.AbiDetect found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.AsyncFFmpegExecuteTask found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.AsyncFFprobeExecuteTask found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.AsyncGetMediaInformationTask found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.BuildConfig found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.CameraSupport found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
> Duplicate class com.arthenica.mobileffmpeg.Config$1 found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.ExecuteCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.FFmpeg found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.FFmpegExecution found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.FFprobe found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.GetMediaInformationCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.Level found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.LogCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.LogMessage found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.MediaInformation found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
> Duplicate class com.arthenica.mobileffmpeg.MediaInformationParser found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.Packages found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.Signal found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.Statistics found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.StatisticsCallback found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.StreamInformation found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.util.Pair found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
>   Duplicate class com.arthenica.mobileffmpeg.util.Trio found in modules jetified-mobile-ffmpeg-full-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-full:4.4.LTS) and jetified-mobile-ffmpeg-min-gpl-4.4.LTS-runtime.jar (com.arthenica:mobile-ffmpeg-min-gpl:4.4.LTS)
> Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
> 
> 
> * Try:
> Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> BUILD FAILED in 31m 56s
> Exception: Gradle task assembleDebug failed with exit code 1
> Exited (sigterm)
> ```
> 
> I have followed already the installation steps on Android

**Comment by seel-channel** [comment]
Are you using a emulator?

**Comment by LeGoffMael** [comment]
Fix provided in #112
